controllers/machine_controller.go

```diff
@@ -235,11 +235,14 @@ func patchMachine(ctx context.Context, patchHelper *patch.Helper, machine *clust
 func (r *MachineReconciler) reconcile(ctx context.Context, cluster *clusterv1.Cluster, m *clusterv1.Machine) (ctrl.Result, error) {
 	log := ctrl.LoggerFrom(ctx)
 
+	res := ctrl.Result{}
 	if conditions.IsTrue(cluster, clusterv1.ControlPlaneInitializedCondition) {
 		if err := r.watchClusterNodes(ctx, cluster); err != nil {
 			log.Error(err, "error watching nodes on target cluster")
 			return ctrl.Result{}, err
 		}
+	} else {
+		res.RequeueAfter = 20 * time.Second
```

Comment:
Why is this needed? The machine controller watches all Cluster objects and enqueue notifications for each Machine in a cluster if needed https://github.com/kubernetes-sigs/cluster-api/pull/5884/files#diff-871c17903b2fd6d91b1f8e5dd5704e6788c0f47131faaa5800ea946fd5ceff40R107